src/redux/books/books.js

```diff
@@ -1,36 +1,49 @@
+import { deleteData, getData, sendData } from './fetchBooks';
+
 const ADD_BOOK = 'booksStore/books/ADD_BOOK';
 const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
-const initialState = [
-  {
-    title: 'Book 1',
-    author: 'Author 1',
-    id: 1,
-  },
-  {
-    title: 'Book 2',
-    author: 'Author 2',
-    id: 2,
-  },
-  {
-    title: 'Book 3',
-    author: 'Author 3',
-    id: 3,
-  },
-];
-
-export const addBook = (payload) => ({
-  type: ADD_BOOK,
+const FETCH_BOOKS = 'bookstore/books/FETCH_BOOKS';
+
+const URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/CZDXQb2zMgHNbimDk0mO/books';
+
+const initialState = [];
+
+export const fetchBooks = (payload) => ({
+  type: FETCH_BOOKS,
   payload,
 });
 
```

Comment:
- [ ] According to the project [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/connec_to_api.md),  " `Use fetch or axios with createAsyncThunk for making API calls` ", please kindly refactor your code to use `createAsyncThunk` for creating thunk action creators. Something like so: 

```
import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
import axios from 'axios'; 


export const getBooks = createAsyncThunk(
  'books/getBooks',
  async () => {
    const response = await axios.get(
      'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/',
    );
    return response.data;
  },
); 
```
Please kindly fix that.